SwrveSDK/src/google/java/com/swrve/sdk/config/SwrveConfig.java

```diff
@@ -62,4 +67,18 @@ public boolean isGAIDLoggingEnabled() {
     public void setGAIDLoggingEnabled(boolean enabled) {
         this.gAIDLoggingEnabled = enabled;
     }
+
+    /**
+     * @return if it will automatically obtain the Google Cloud Messaging Registration ID.
+     */
+    public boolean isAutomaticPushRegistration() {
```

Comment:
Apologies again for being pedantic. But if we call the flag pushRegistrationAutomatic, then the method can be called isPushRegistrationAutomatic.......which is the nicest! :) Feel free to ignore this comment though.